bazel/foreign_cc/luajit.patch

```diff
@@ -11,15 +11,6 @@ index e65b55e..f0a61dd 100644
  #
  # Use this if you want to force a 32 bit build on a 64 bit multilib OS.
  #CC= $(DEFAULT_CC) -m32
-@@ -36,7 +36,7 @@ CC ?= $(DEFAULT_CC)
- # to slow down the C part by not omitting it. Debugging, tracebacks and
- # unwinding are not affected -- the assembler part has frame unwind
- # information and GCC emits it where needed (x64) or with -g (see CCDEBUG).
--CCOPT= -O2 -fomit-frame-pointer
-+CCOPT= -O2 -fomit-frame-pointer -fPIC -fPIE -fplugin=annobin -fstack-protector-strong -fstack-protector-all
```

Comment:
question: this looks like annobin & at least some of the hardening flags work with annobin?

Reply:
yeah, for now I'm removing everything. We can work towards re-enabling them with the possible exception of annobin.